File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsExperimentalFeaturesFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsExperimentalFeaturesFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsExperimentalFeaturesFragment().getTitleResourceId()).isEqualTo(R.string.experimental_features);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsExperimentalFeaturesFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_experimental_features);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsExperimentalFeaturesFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsExperimentalFeaturesFragmentTest {
	@Test
	fun getTitleResourceId() {
		assertThat(SettingsExperimentalFeaturesFragment.newInstance().getTitleResourceId()).isEqualTo(R.string.experimental_features)
	}

	@Test
	fun getXmlResourceFile() {
		assertThat(SettingsExperimentalFeaturesFragment.newInstance().getXmlResourceFile()).isEqualTo(R.xml.settings_experimental_features)
	}
}
